.github/scripts/check_docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+dest=$(pwd)/.scala/bin
+
+./mill -i copyTo cli.launcher $dest/scala-cli 
+
+export PATH=$dest:$PATH
+echo Adding $dest to classpath
+ls $dest
+
+./mill -i scala docs_checker/check.scala -- docs/cookbooks 
```

Comment:
We could make this whole file a Mill command, with something along those lines in `build.sc`:
```scala
def checkDocs(args: String*) = T.command {
  val launcher = cli.launcher().path
  os.proc(launcher, "docs_checker/check.scala", if (args.isEmpty) Seq("docs/cookbooks") else args)
    .call(stdout = os.Inherit)
}
```

Run with:
```text
$ ./mill -i checkDocs
```

Watch mode should work for the Scala CLI launcher itself:
```text
$ ./mill -i -w checkDocs
```

With some clever use of Mill `T.sources`, we could even get watch mode to work for the cookbook themselves. Maybe with something like
```scala
def cookbookSources = T.sources {
  Seq(PathRef(os.pwd / "docs" / "cookbooks"))
}
def checkDocs() = T.command {
  val launcher = cli.launcher().path
  val sources = cookbookSources().map(_.path)
  os.proc(launcher, "docs_checker/check.scala", sources)
    .call(stdout = os.Inherit)
}
```